pkg/splunk/enterprise/afwscheduler.go

```diff
@@ -1229,6 +1233,8 @@ func getClusterScopePlaybookContext(ctx context.Context, client splcommon.Contro
 	switch kind {
 	case "ClusterMaster":
 		return getIdxcPlaybookContext(ctx, client, cr, afwPipeline, podName, podExecClient)
+	case "ClusterManager":
```

Comment:
Probably these 2 cases can be clubbed into one since both call same playbook context?

Reply:
I didn't know it could be done in Golang. Updating this in the next commit.

Reply:
Done